### PR TITLE
Added docs for script checks

### PR DESCRIPTION
**Pull request by diptanu**
While I was writing the docs, I started thinking that it might be useful if we supported string interpolation of the command and args for script checks. Users might be able to pass, host name, version of software, etc, as arguments to the script. Thoughts? cc/ @dadgar @mitchellh @pearkes

### CODE REVIEW

File: website/source/docs/jobspec/servicediscovery.html.md [hunk 1]
```diff
@@ -61,6 +61,14 @@ group "database" {
                 interval = "10s"
                 timeout = "2s"
             }
+            check {
+                type = "script"
+                name = "check_table"
+                cmd = "/usr/local/bin/check_mysql_table_status"
+                agrs = ["--verbose"]
```
Comment (adrianlop) [inline website/source/docs/jobspec/servicediscovery.html.md]: typo: args

File: website/source/docs/jobspec/servicediscovery.html.md [hunk 2]
```diff
@@ -120,9 +130,14 @@ group "database" {
...
 
+* `cmd`: This is the command that the Nomad client runs for doing script based
+  health check.
+
+* `args`: Additional arguments to the `cmd` for script based health checks.
+
 ## Assumptions
 
-* Consul 0.6.0 or later is needed for using the TCP checks.
+* Consul 0.6.4 or later is needed for using the TCP checks.
```
Comment (adrianlop) [inline website/source/docs/jobspec/servicediscovery.html.md]: I think TCP was already working on 0.6:
* Consul 0.6.0 or later is needed for using the TCP checks.
* Consul 0.6.4 or later is needed for using the script checks.

Comment (diptanu) [inline website/source/docs/jobspec/servicediscovery.html.md]: @adrianlop Thanks!

**Comment by mitchellh** [comment]
I don't see why not.

**Comment by diptanu** [comment]
@mitchellh Done!

**Pull request merged**